doc/python/px-arguments.md

```diff
@@ -244,6 +244,8 @@ fig = px.density_contour(dict(effect_size=5 + np.random.randn(N),
 fig.show()
 ```
 
+To pass a `dict` or an array (such as a NumPy `ndarray`) to the `data_frame` argument, you'll need to have pandas installed, because `plotly.express` internally converts the `dict` or array to a pandas DataFrame.
```

Comment:
```suggestion
To pass a `dict` or an array (such as a NumPy `ndarray`) to the `data_frame` parameter, you'll need to have pandas installed, because `plotly.express` internally converts the `dict` or array to a pandas DataFrame.
```